tests/lib/tools/tests.backup

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh -e
+
+show_help() {
+    echo "usage: tests.backup prepare"
+    echo "       tests.backup restore"
+}
+
+cmd_prepare() {
+    tar cf "${PWD}.tar" "$PWD"
+}
+
+cmd_restore() {
+    if [ -z "$PWD" ] || [ "$PWD" != "$(pwd)" ]; then
```

Comment:
I would not do this. Instead the tool should take an optional path to operate on, which defaults to pwd.

Reply:
sure, I left [ -z "$PWD" ] because thanks to that I found a problem and had to change the way we use to clean the directory

Reply:
done